errors/errors.go

```diff
@@ -7,6 +7,7 @@ var (
 	ErrCliBadConfig            = errors.New("cli: bad config")
 	ErrRepoNotFound            = errors.New("repository: not found")
 	ErrRepoIsNotDir            = errors.New("repository: not a directory")
+	ErrCVENotFound             = errors.New("CVE: no CVE found for image")
```

Comment:
We shouldn't have such an error. Having no CVEs is a good thing.
And I also think this should be handled on ZLI side. It's a matter of how we present the results to the user.